x-pack/legacy/plugins/apm/public/components/app/Permission/index.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiTitle
+} from '@elastic/eui';
+import React, { useMemo, useState } from 'react';
+import { matchPath } from 'react-router-dom';
+import styled from 'styled-components';
+import { useLocation } from '../../../hooks/useLocation';
+import { pct, px, unit } from '../../../style/variables';
+import { SetupInstructionsLink } from '../../shared/Links/SetupInstructionsLink';
+import { routes } from '../Main/route_config';
+import { useFetcher, FETCH_STATUS } from '../../../hooks/useFetcher';
+
+export const Permission: React.FC = ({ children }) => {
+  const [isPermissionPageEnabled, touglePermissionPage] = useState(true);
```

Comment:
Should be `togglePermissionPage`.

Reply:
...and I prefer (the admittedly longer) `setIsPermissionsPageEnabled`, since it make it easier to see that these are get/set.